### DISCUSSION TITLE
Attempt to call function pcam_host_added in base 'null instance' on a null instance

### DISCUSSION THREAD

**Discussion by ferolschoon** [ramokz/phantom-camera#419] [Question / Help Wanted]
I am trying to use the PhantomCamera3D in a scene.  Everytime I start up my scene it gets the following error: :"Attempt to call function pcam_host_added in base 'null instance' on a null instance".  I have tried a few different setups and all give me this error.  Currently Im just trying to get it working on an empty scene with nothing but a PhantomCamera3D and Camera3D with a child PhantomCameraHost.  I have followed a handful of youtube guides on how to set this up, and I run into this issue each time despite following their instructions.
![sceneSetup](https://github.com/user-attachments/assets/dac14ebe-9709-4038-81d9-05e1a11004d4)

**Comment by ramokz** [comment]
The scene set up in the screenshot is correct, so there could be a fair few reasons behind the issue.

A couple of quick checks, as it's difficult to tell much from the image alone:
- Are you using the latest version?
- Have you enabled the addon inside Project Settings
- Do you have any other things running in the project?

**Comment by galih9** [comment]
I can verify that you have to enable the plugin from the project settings, im using godot 4.6, you can found it in the plugin tab


<img width="749" height="233" alt="Screenshot 2026-02-03 at 13 43 32" src="https://github.com/user-attachments/assets/e214a829-ec3a-4c1b-a56b-b48991dc9bb4" />
